README.md

```diff
@@ -64,6 +64,8 @@ Thanks a lot to all the previous [contributors](https://github.com/i18next/i18ne
 - **-l, --locales <list>**: The locales in your applications. Defaults to `en,fr`
 - **--directoryFilter**: Globs of folders to filter
 - **--fileFilter**: Globs of files to filter
+- **--keep-removed**: Prevent keys no longer found from being removed
+- **--no-old**: Avoid saving the _old files
```

Comment:
I prefer we don't multiply the APIs. There is already a `writeOld` option doing the same thing.

For backward compatibility and consistency, can you make that `writeOld`?